tests/test_api/test_users.py

```diff
@@ -82,7 +82,3 @@ def test_delete_user(self, test_client, user):
         check_valid_header_type(delete.headers)
         data = json.loads(delete.data)
         assert data['result'] == True
-
-        # Dummy user should no longer be logged in
```

Comment:
All this checks now is whether login_required is working, which I made as a separate test (above)